src/EnergyPlus/EMSManager.cc

```diff
@@ -728,8 +656,7 @@ namespace EMSManager {
                                                                              lAlphaFieldBlanks,
                                                                              cAlphaFieldNames,
                                                                              cNumericFieldNames);
-                } else if (ActuatorNum > state.dataRuntimeLang->numActuatorsUsed &&
-                           ActuatorNum <= state.dataRuntimeLang->numActuatorsUsed + state.dataRuntimeLang->NumExternalInterfaceActuatorsUsed) {
+                } else if (ActuatorNum <= state.dataRuntimeLang->numActuatorsUsed + state.dataRuntimeLang->NumExternalInterfaceActuatorsUsed) {
```

Comment:
This one, and onother further down, is CppCheck correction.

    [src/EnergyPlus/EMSManager.cc:731]:(style),[knownConditionTrueFalse],Condition 'ActuatorNum>state.dataRuntimeLang->numActuatorsUsed' is always true
    [src/EnergyPlus/EMSManager.cc:1114]:(style),[knownConditionTrueFalse],Condition 'ActuatorNum>state.dataRuntimeLang->numActuatorsUsed' is always true